### DISCUSSION TITLE
[绳网典型问题]《人工》智能回答区

### DISCUSSION THREAD

**Discussion by Buer-Nahida** [share121/inter-knot#1631] [其他]
![图片](https://github.com/user-attachments/assets/f1490d4c-ac69-4070-858c-cd81591e65ab)

什么问题都可以问，我会不定期看下这个帖子，回答下问题（反正又没多少人评论，我干脆`Subscribe`这条帖子了，不怕邮箱轰炸）

- [ ] x

**Comment by inkluv** [comment]
我的Replay软件死活找不到我的3060Laptop，只能用cpu来推理，怎么解决（）
Replay用的UVR5模型分离人声，RVC模型变声（应该是

**Reply by Buer-Nahida** [reply]
容我研究下RVC，我现在用的SVC，RVC这东西我是一点也不知道

**Reply by Buer-Nahida** [reply]
RVC好像是默认使用GPU的吧，找不到GPU的是UVR5还是RVC啊？

**Reply by Buer-Nahida** [reply]
你使用的Windows还是Linux，Linux的话应该是Nvidia显卡驱动没配置好

**Reply by inkluv** [reply]
win11，好像是这个软件找不到，这个软件是一键分离人声然后自动变声，最后在把音轨拼接起来（大概是这样），输出变声过后的歌曲。
我英伟达面板和windows设置都给这个软件选择高性能，好像没啥用

**Reply by Buer-Nahida** [reply]
额，你说的软件是这个[RVC-Project/Retrieval-based-Voice-Conversion-WebUI](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI)项目吗？

**Comment by capacitor1** [comment]
提问：楼主为什么不是Collaborator

**Reply by Buer-Nahida** [reply]
能力越大，责任越大。~~压力山大~~

**Reply by capacitor1** [reply]
> ~能力越大~责任越大

好好好
